src/views/index/index.js

```diff
@@ -2,14 +2,16 @@ import Vue from 'vue'
 
 import { installServiceWorker } from '../../serviceWorkerInstaller'
 
-const ButtonGroup = () => import(/* webpackChunkName: "example" */ '../../components/Button')
-const CheckboxGroup = () => import(/* webpackChunkName: "example" */ '../../components/Checkbox')
-const ProgressBar = () => import(/* webpackChunkName: "example" */ '../../components/ProgressBar')
-const RadioButtons = () => import(/* webpackChunkName: "example" */ '../../components/RadioButtons')
+const ButtonGroup = () => import(/* webpackChunkName: "example" */ '../../components/Button/index')
```

Comment:
why did you choose to split some components into three files? 
'Cause of the file size?

Reply:
Yes, for that reason.